pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -528,6 +529,13 @@ type PDFailureMember struct {
 	CreatedAt     metav1.Time `json:"createdAt,omitempty"`
 }
 
+// UnjoinedMember is the pd unjoin cluster member information
+type UnjoinedMember struct {
```

Comment:
Instead of add an attribute to `TidbCluster`, does just emit a `UnjoinedMember` and `JoinedMember` events to `TidbCluster` make sense?

User can use `kubectl describe` to get these messages easily.

Reply:
I think event flow can't express my meaning in a concise manner , as a user, I need to understand and integrate the event flow and sort out each step. Use `status` type  display current status is a concise way.